cloud_sql/sqlserver_instance_pitr/main.tf

```
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_sqlserver_instance_pitr]
+resource "google_sql_database_instance" "default" {
+  name             = "sqlserver-instance-pitr"
+  region           = "asia-northeast1"
+  database_version = "SQLSERVER_2019_ENTERPRISE"
```

CI is failing due to ` Error 400: Missing parameter: Root Password., required`. This is required for MSSQL. 
```suggestion
  database_version = "SQLSERVER_2019_ENTERPRISE"
  root_password    = "INSERT-PASSWORD-HERE"
```